assets/templates/reports-filters.html

```diff
@@ -120,6 +120,21 @@ <h6 class="subtitle-kpi">contactos nuevos</h6>
         </ul>
       </div><!-- reports-box -->
     </div>
+    <div class="wrapper-ranking">
+      <div class="reports-box">
+        <small class="title-ranking">TOP PAGINAS MAS VISITADAS</small>
```

Comment:
Could you avoid uppercase text and use styles in cases like this?

Reply:
Ok, I correct